Doc/howto/regex.rst

```diff
@@ -245,6 +245,9 @@ You can omit either *m* or *n*; in that case, a reasonable value is assumed for
 the missing value.  Omitting *m* is interpreted as a lower limit of 0, while
 omitting *n* results in an upper bound of infinity.
 
+The simplest case ``{m}`` matches the preceding item exactly m times.
```

Comment:
Let's put "m" in italics to match the previous paragraphs:

```suggestion
The simplest case ``{m}`` matches the preceding item exactly **m** times.
```

Reply:
Oops, this should have been `*m*` for reST! Would you like to open a PR to fix it?